server.php

```diff
@@ -1,21 +0,0 @@
-<?php
-
-/**
- * Laravel - A PHP Framework For Web Artisans
- *
- * @package  Laravel
- * @author   Taylor Otwell <taylor@laravel.com>
- */
-
-$uri = urldecode(
-    parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH)
-);
-
-// This file allows us to emulate Apache's "mod_rewrite" functionality from the
-// built-in PHP web server. This provides a convenient way to test a Laravel
-// application without having installed a "real" web server software here.
-if ($uri !== '/' && file_exists(__DIR__.'/public'.$uri)) {
-    return false;
-}
-
-require_once __DIR__.'/public/index.php';
```

Comment:
Why was this file deleted? I think it is being used when you use the command `php artisan serve`

Reply:
I am using Laragon, so I think Avast made some problems, it may treat the file server.php as a threat and automatically deleted it. I added it to the exclusions, so it should fix the problem for future work.

Reply:
ok cool, I'll bring it back